library/custom_template/ajax_code.php

```diff
@@ -50,12 +50,12 @@
             sqlStatement("INSERT INTO template_users (tu_user_id,tu_template_id) VALUES (?,?)", array($_SESSION['authUserID'], $newid));
         }
         echo "<select name='template' id='template' onchange='TemplateSentence(this.value)' style='width:180px'>";
-        echo "<option value=''>" . htmlspecialchars(xl('Select category'), ENT_QUOTES) . "</option>";
+        echo "<option value=''>" . xlt('Select category') . "</option>";
```

Comment:
needs to be xla() option value is an attribute.

Reply:
Thanks! I was too lazy to do this script!:)

Reply:
lol, look at me Brady, I'm catching escaping missteps.

Reply:
![2DV](https://github.com/openemr/openemr/assets/278968/5e30d176-555f-4546-9e80-760d537f60d9)